extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -53,6 +55,54 @@
   opacity: 0 !important;
 }
 
+.i-amphtml-story-story-attribution {
+  top: 0 !important;
+  border: none !important;
+  pointer-events: auto !important;
+  cursor: pointer !important;
+  padding: 6px 3px 0 !important;
+  height: 48px !important;
+  place-items: center !important;
+  display: flex !important;
+  text-decoration: none !important;
+  overflow: hidden !important;
+  visibility: hidden !important;
+}
+
+[desktop] .i-amphtml-story-story-attribution {
+  padding: 6px 6px 0 !important;
+  left: 0 !important;
+  right: 0 !important;
+}
+
+.i-amphtml-story-story-attribution > * {
+  margin: 0px 3px !important;
+}
+
+.i-amphtml-story-attribution-logo-container {
+  display: grid !important;
+  place-items: center !important;
+}
+
+.i-amphtml-story-attribution-logo {
```

Comment:
nit:
We might want to put a background color on these to help with contrast on transparent PNGs:
<img width="533" alt="Screen Shot 2021-04-02 at 10 52 40 AM" src="https://user-images.githubusercontent.com/3860311/113426563-a60fc080-93a1-11eb-88bb-bd41c435d047.png">
With # d4d4d4 bg:
<img width="539" alt="Screen Shot 2021-04-02 at 10 52 31 AM" src="https://user-images.githubusercontent.com/3860311/113426600-b1fb8280-93a1-11eb-9de2-c4ca0630739b.png">

Reply:
Good catch :)

Reply:
Wait, I think this would mess up some logos that expect the transparency to be there. e.g.

Actual: 
![image](https://user-images.githubusercontent.com/5449100/113439044-9cd12480-939e-11eb-84d5-70b2339aa4fb.png)

Expected: 

![image](https://user-images.githubusercontent.com/5449100/113439070-a8bce680-939e-11eb-9992-12eb034d6d8d.png)


I think for now we should not add a background-color. With the option for later exposing a background-logo-color that publishers could use to add whichever color they like?